packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -16,6 +16,8 @@
   "force_source_status_text":
     "SFDX: View All Changes (Local and in Default Scratch Org)",
   "force_apex_test_run_text": "SFDX: Invoke Apex Tests...",
+  "force_apex_test_run_class_text": "SFDX: Invoke Last Executed Test Class",
```

Comment:
@ruthemmanuelle - These messages were updated since the last time you reviewed based on the conversation at https://github.com/forcedotcom/salesforcedx-vscode/pull/308#issuecomment-366398242. Could you take another look and see if they are OK? Thanks!

Reply:
Are we removing the other code actions (`SFDX: Invoke Apex Test Method` and `SFDX: Invoke All Apex Tests in Class`)? If so, I'd rather move away from using "invoke," even though `SFDX: Invoke Apex Tests` uses it. I think that "run" is more intuitive than "invoke," and "re-run" is clearer than "invoke" in this case.

"SFDX: Re-Run Last Executed Apex Test Class"
"SFDX: Re-Run Last Executed Apex Test Method"

Reply:
@ruthemmanuelle - Correct, we are removing the others. There is no more  `SFDX: Invoke Apex Test Method` or `SFDX: Invoke All Apex Tests in Class`.

In that case, let's change it to `Re-Run`.

Reply:
Ok. Please note that the versions in my comment also added "Apex" to the messages.

https://github.com/forcedotcom/salesforcedx-vscode/tree/develop/packages/salesforcedx-vscode-core#run-apex-tests will need to be updated as a result of this change, and we'll need to remove or update https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/develop/packages/salesforcedx-vscode-core/images/apex_test_run.gif . I should be able to do the former in time for this week's release, but I might not have time to do the latter.

Reply:
@ruthemmanuelle Actually. Let me be clearer since I think there is a potential for confusion.  

There is no top-level command, but what you wrote in the README.md still holds. Basically, the lightbulbs still use the name SFDX: Invoke Apex Test Method and SFDX: Invoke All Apex Test in Class. But, when you use CMD + P, you won't be able to see those methods.

```
## Run Apex Tests
To run Apex tests, in your `.cls` file, click the name of an Apex test class or method. Next, click the code actions lightbulb that appears by the class or method signature, and then select **SFDX: Invoke Apex Test Method** or **SFDX: Invoke All Apex Tests in Class**.  
![GIF that shows running Apex tests for a method and then a class](https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/develop/packages/salesforcedx-vscode-core/images/apex_test_run.gif)

```